template/interface_cpp.njk

```diff
@@ -38,12 +38,26 @@ void {{name}}Bridge::Init(Napi::Env env, Napi::Object exports) {
 
 {{name}}Bridge::{{name}}Bridge(const Napi::CallbackInfo& info)
     : Napi::ObjectWrap<{{name}}Bridge>(info) {
-  // TODO: implement constructor overloading
-  {% for idl_constructor in constructors %} new {{name}} (
-    {% for argument in idl_constructor.arguments %}
-      NativeTypeTraits<IDL{{argument.type | camelcase-}}>::NativeValue(info.Env(), info[{{loop.index - 1}}]){% if not loop.last %},{% endif %}
-    {% endfor %})
-  {% endfor %};
+  if ({{constructors.length}} == 0 && info.Length() == 0) {
+    new {{name}} ();
+    return;
+  }
+
+  {% for idl_constructor in constructors %} 
+    if (info.Length() == {{idl_constructor.arguments.length}}
```

Comment:
Please use a variable instead of calling info.Length() repeatedly.
If the argument's length is long, the call overhead can be increased.

Reply:
Okay I will change